src-tauri/Cargo.lock

```diff
@@ -7431,8 +7437,9 @@ dependencies = [
 
 [[package]]
 name = "tauri"
-version = "2.1.1"
-source = "git+https://github.com/tari-project/tauri.git?rev=67a06c8a9bae94f412b8059dfa8b4d8dd8ea0a25#67a06c8a9bae94f412b8059dfa8b4d8dd8ea0a25"
+version = "2.2.2"
```

Comment:
This concerns me, because on CI it was updating to tauri 2.2.2 which caused builds to break. 

So CI is locked to 2.1.1. Which is messy, as it's not what we're all running etc.

Reply:
CI looks like it ran fine since that's where the install happens - https://github.com/tari-project/universe/actions/runs/12768650543/job/35589473800?pr=1379

also this it just `tauri` not `tauri-cli` :D + on `main` the cargo.lock includes `tauri-plugin-cli` at `2.2.0` anyway 😅